Magento_CompanyPayment/templates/company/management/profile/payment-method-list.phtml

```diff
@@ -5,17 +5,31 @@
 ?>
 
 <div class="dashboard-items">
-    <h2 class="dashboard-items__title margin-bottom-sm">
+    <h2
+        class="
+            dashboard-items__title
+            margin-bottom-sm
+        "
+    >
```

Comment:
FYI, 2 classes in one line are ok so this change isn't necessary but it's fine too.